src/gameFunctionality/flipCard.ts

```diff
@@ -80,8 +80,30 @@ export const flipCard = (clue: string, socket: Socket) => {
                 }
             }
         }
-        socket.emit("updateGameProperties", {
-            gameArray: gameArray
-        } as gamePropertiesObj)
-    }
+        socket.emit("updateGameProperties", {gameArray: gameArray} as gamePropertiesObj)
+        console.log("nextTurn:", nextTurn)
+        }
+        if (gamePropertiesStore.firstTeamWords?.includes(clue)){
+            const indexOfTheWord = gamePropertiesStore.firstTeamUnguessedWords?.indexOf(clue);
+            if (indexOfTheWord !== -1 && typeof indexOfTheWord === "number") {

```

Comment:
typeof indexOfTheWord === "number" 
is it necessary?

Reply:
fixed